src/Sylius/Behat/Context/Ui/Admin/ManagingLocalesContext.php

```diff
@@ -66,6 +66,7 @@ public function __construct(
 
     /**
      * @Given I want to create a new locale
```

Comment:
This should be removed and the below step should be used everywhere. Task for a separate PR.